cmd/mr_discussion.go

```diff
@@ -117,6 +132,7 @@ func mrDiscussionText() (string, error) {
 func init() {
 	mrCreateDiscussionCmd.Flags().StringSliceP("message", "m", []string{}, "use the given <msg>; multiple -m are concatenated as separate paragraphs")
 	mrCreateDiscussionCmd.Flags().StringP("file", "F", "", "use the given file as the message")
+	mrCreateDiscussionCmd.Flags().StringP("commit", "", "", "start a thread on a commit")
```

Comment:
It's missing the same "-c" option in the `StringP()`.

Reply:
Oops.  Fixed there too!